Dockerfile

```diff
@@ -0,0 +1,4 @@
+# Initialise a base image
+FROM python:3.7-stretch
```

Comment:
The base will be a Jina installation, probably: [jina:master-standard](https://hub.docker.com/layers/jinaai/jina/master-standard/images/sha256-32178a37712ef0c6219a9b6fb39cd61a17a0942583a264786137a4e32fd09bf9?context=explore)

Reply:
```suggestion
FROM jina:master-standard
```